how-to-guides/consensus-node.md

```diff
@@ -219,10 +219,16 @@ a trusted RPC endpoint (/block). RPC endpoints are also required for retrieving
 light blocks. These can be found in the docs here under the respective networks or
 from [the chain-registry](https://github.com/cosmos/chain-registry).
 
-In `$HOME/.celestia-app/config/config.toml`, set
+::: warning
+State sync requires at least two RPC endpoints to function. These endpoints are used to:
```

Comment:
Taking a step back, what's the benefit of state sync over snapshot sync? I know they're both options, but we don't have to actually document and support both options. Snapshot sync just looks easier to set up and more reliable since it decouples downloading the snapshot from running your node. cc @evan-forbes for discussion